src/nginxconfig/generators/conf/security.conf.js

```diff
@@ -58,13 +58,186 @@ export default (domains, global) => {
         config.push(['location /security.txt', {
             return: '301 /.well-known/security.txt',
         }]);
-        
+
         // Custom security.txt path
         config.push(['location = /.well-known/security.txt', {
             alias: `${global.security.securityTxtPath.value}`,
         }]);
     }
 
+    if (global.security.blockCommonExploits.computed) {
+        // Block SQL Injections
+        config.push(['## Block SQL injections', '']);
```

Comment:
```suggestion
        config.push(['# Block SQL injections', '']);
```
Consistent use of single `#` for comments please (apply the same feedback to the others below as well)